### ISSUE TITLE
AWS organizaiton

### ISSUE THREAD

**Issue by camcarlton** [CodeSpaceHQ/Telescreen#34]
First, do we want to store any face metadata?
This metadata can consist of whether or not a citizen has glasses, sunglasses, whether or not they are smiling, or showing other emotions.

Second, sooner or later I am going to need to know if we want to upload images to S3 from the telescreens or try to have them as raw images. 
The connection to any Amazon Service is not instant but it isn't slow by no means. The benefit of having the image upload to S3 is that all devices will have access to the image for processing or for reference. Downside as of now is I am unsure how fast/slow the upload would be and we have 5 GB of Amazon S3 storage, 20,000 Get Requests, 2,000 Put Requests, and 15 GB of data transfer out each month for one year.

Lastly, do we want to have bash or other scripts language files to setup and configure AWS on any device?
Basically installing the AWS portion for any possible telescreen. If so should each device be considered a user in the IAM or just one user for all the telescreens?

**Comment by camcarlton** [comment]
Will not store the face metadata.

Sending pictures straight to S3 through a script. Pictures will be sent as jpg.

We will need a bash script towards the end of our project.
